src/libnnstreamer-edge/nnstreamer-edge-custom-impl.h

```diff
@@ -0,0 +1,85 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * Copyright (C) 2024 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * @file   nnstreamer-edge-custom-impl.h
+ * @date   21 Aug 2024
+ * @brief  Internal interface to support communication using custom library.
+ * @see    https://github.com/nnstreamer/nnstreamer
+ * @author Gichan Jang <gichan2.jang@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+
+#ifndef __NNSTREAMER_EDGE_CUSTOM_IMPL_H__
+#define __NNSTREAMER_EDGE_CUSTOM_IMPL_H__
+
+#include "nnstreamer-edge-custom.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+/**
+ * @brief Data structure for edge custom connection.
+ */
+typedef struct
+{
+  void *dl_handle;
+  void *instance;
+  void *priv;
+} custom_connection_s;
+
+/**
+ * @brief Internal function to create custom connection.
+ */
+int nns_edge_custom_create (custom_connection_s *custom, const char *lib_path);
```

Comment:
Isn't it confusing for users to use the same function name with nnstreamer-edge-custom.h?

Reply:
updated function name to _load()